R/clean_names.R

```diff
@@ -24,6 +24,7 @@
 #' (characters) to "u".
 #'
 #' @param dat The input `data.frame`.
+#' @param set_labels If set to `TRUE`, old names are stored as labels in each column of `dat`.
```

Comment:
Please modify to "... in each column of the returned data.frame."

Reply:
Done